pennylane/ops/op_math/adjoint_class.py

```diff
@@ -224,3 +232,21 @@ def diagonalizing_gates(self):
 
     def adjoint(self):
         return self.base.queue()
+
+    @property
+    def arithmetic_depth(self) -> int:
+        return 1 + self.base.arithmetic_depth
+
+    def simplify(self, depth=-1):
+        # TODO: Should we add a self.do_queue attribute and instantiate the simplified class
+        # with the same value?
+        if depth == 0:  # TODO: We could add a decorator to do this check for all simplify methods
+            return self
+        # TODO: Should we do: Adj(AB).simplify() = Adj(B) Ajd(A) ?
+        if isinstance(self.base, Adjoint) and (depth > 1 or depth < 0):  # Adj(Adj(A)) = A
+            return self.base.base.simplify(depth=depth - 2)
+        if isinstance(self.base, Sum):  # Adj(A + B) = Adj(A) + Adj(B)
+            sim_sum = self.base.simplify(depth=depth)
```

Comment:
```suggestion
            simplified_sum = self.base.simplify(depth=depth-1)
```

Reply:
Should we add the `-1` here? If the depth is defined as the maximum number of stacked operators, when we do `Adj(Sum(A, B)) --> Sum(Adj(A), Adj(B))` we are not reducing the depth. That is why I didn't add the `-1`.

By adding the `-1` here then we would have inconsistencies such as:
```python
>>> import pennylane as qml
>>> adj_op = qml.adjoint(qml.op_sum(qml.op_sum(qml.PauliX(0), qml.PauliX(0)), qml.PauliX(0)))
>>> adj_op.arithmetic_depth
3
>>> sum_op = adj_op.simplify(depth=1)
>>> sum_op.arithmetic_depth
3
```